Detectors/TOF/workflow/CMakeLists.txt

```diff
@@ -43,3 +44,7 @@ o2_add_executable(cluster-calib
                   COMPONENT_NAME tof
                   PUBLIC_LINK_LIBRARIES O2::TOFWorkflowUtils)
 
+o2_add_executable(cluscal-reader
```

Comment:
could you change the name to `cluscal-reader-workflow`, to differentiate it from self-contained executables?

Reply:
Yes,
I applied the same change to cluster-calib-workflow